user/login.php

```diff
@@ -11,7 +11,30 @@ public static function run(){
     //Start een connectie
     MysqlHelper::startConnection();
 
-    echo $_POST["username"]; // echo de data array in json formaat voor de frontend
+    //Doe dingen
+    $email = $_POST['email'];
+    $password = $_POST['password'];
+
+    $user = UserModel::getByEmail($email);
+    if (!isset($user)){
+      echo "incorrect";
+      return false;
+    }
+    $response = $user->checkPassword($password);
+    if ($response === true){
+      echo "Succes!";
+      $_SESSION['email'] = $POST['user'];
+      http_response_code(200);
+    }
+    else if ($response === false ){
+      http_response_code(400);
+      echo "incorrect password";
```

Comment:
Hier alleen incorrect, omdat je vanwege security reasons eig niet wil vrijgeven wat er precies fout is.